dataGenerator.js

```diff
@@ -23,149 +23,940 @@ var db = require('./server/db.js');
 var users = require('./server/users.js');
 var questions = require('./server/questions.js');
 var common = require('./server/common.js');
+var datagenInfo = require('./datagenInfo.js');
 
 // variables to control the genereated data
+
```

Comment:
extraSpace.html

Reply:
dkm, fixed

Reply:
this is no laughing matter. This is serious

Reply:
also can we add animation

Reply:
consider it done

Reply:
nice work fam. I knew I can always count on you